ui/app/styles/core/forms.scss

```diff
@@ -21,7 +21,7 @@ label {
 .b-checkbox .is-label {
   color: $grey-darker;
   display: inline-block;
-  font-size: $size-small;
+  font-size: 14px;
```

Comment:
Why are we deviating from the variables here?

Reply:
Yep, we're deviating.  I walked through Ivana on this as I noticed her designs were different than what we currently have and. she asked me to update all.